include/gdsync/core.h

```diff
@@ -57,6 +57,7 @@ enum gds_create_qp_flags {
 };
 
 typedef struct ibv_qp_init_attr_ex gds_qp_init_attr_t;
+typedef struct ibv_qp_attr gds_qp_attr_t;
```

Comment:
Yes, I'm using "struct ibv_qp_attr" without the gds typedef. I can remove it from here.